doc/examples/segmentation/plot_multiotsu.py

```diff
@@ -53,7 +53,7 @@
     ax[1].axvline(thresh, color='r')
 
 # Plotting the Multi Otsu result.
-ax[2].imshow(regions, cmap='Accent')
+ax[2].imshow(regions, cmap='jet')
```

Comment:
I think this change is probably not necessary?

Reply:
I think someone mentioned a color change to lessen the pink effect here ->https://github.com/scikit-image/scikit-image/issues/4993#issuecomment-705545483.
The cmap with jet parameter looks better than Accent.

Reply:
Ah, fair enough. I just have a severe allergy to jet. LOL I think even if not accent we should use one of the **qualitative** colormaps found in [this page](https://matplotlib.org/3.1.1/tutorials/colors/colormaps.html)